spec/features/staff/users_can_mark_a_report_in_review_spec.rb

```diff
@@ -0,0 +1,50 @@
+RSpec.feature "Users can move reports into review" do
+  context "signed in as a BEIS user" do
+    let(:beis_user) { create(:beis_user) }
+
+    before do
+      authenticate!(user: beis_user)
+    end
+
+    scenario "they can mark a report as in review" do
+      report = create(:report, state: :submitted)
+
+      visit report_path(report)
+      click_link I18n.t("table.body.report.action.in_review")
```

Comment:
These tests look solid, we just need to swap `I18n.t` for `t` to check for missing translation keys.